internal/command/command.go

```diff
@@ -107,6 +107,13 @@ func newRunE(fn Runner, preparers ...preparers.Preparer) func(*cobra.Command, []
 		task.FromContext(ctx).Start(ctx)
 
 		sendOsMetric(ctx, "started")
+		task.FromContext(ctx).RunFinalizer(func(ctx context.Context) {
+			err := metrics.FlushMetrics()
+			if err != nil {
+				fmt.Fprintln(os.Stderr, "Error spawning metrics process: ", err)
```

Comment:
You should probably use the `iostreams.StdErr` that already contained in the context.